### DISCUSSION TITLE
Working to Bring More Women and People of Color Into the API Specification Conversation

### DISCUSSION THREAD

**Discussion by kinlane** [OAI/OpenAPI-Specification#2713] [General]
I wanted to start a conversation around the troubling fact that there are too few women and people of color involved at all levels of this community. AsyncAPI’s Senior Technical Writer Alejandra Quetzall brought it up in the SIG call today, but it is something that continues to get brought up, yet continues to be a problem for us—that the API specification conversation is largely dominated by white men. I know that many of us dread having this conversation, but trust me, it is no fun being the person to bring it up (or be affected by it), and I’d rather do the work than require women and people of color to keep doing the hard work here. So I wanted to kick off this conversation around what we can do to address this issue—I am looking for discussion around two things:

- What obstacles currently keep women from participating in conversations within this community?
- What can we do to attract more women to these communities and empower them to participate?

If you aren’t comfortable having this discussion here on the forum, ~~feel free to email your ideas at kinlane@gmail.com~~ _[**EDIT:** Please reach out to a [TSC member](https://github.com/OAI/OpenAPI-Specification/blob/main/MAINTAINERS.md) directly such as through DM on Slack]_, but I would prefer that we have this conversation out in the open, no matter how uncomfortable it makes us. I’d also like to make sure we keep this conversation friendly, inclusive, and supportive, but also make sure it is an honest and straightforward one. I look forward to hear your ideas and thoughts, and if you are someone from any of the underrepresented communities that are regularly absent from technology and the world of APIs, feel free to ping me directly for help learning about how you can get more involved.

**Comment by handrews** [comment]
An update on this (which I am merely reporting and take no credit for whatsoever):  @lornajane and @miqui are now provisional TSC members, and currently the folks who most commonly lead the TDC calls.  @miqui has spoken in a TDC call about recruiting more participation from Latin America, and the TDC calls recently passed the [Bechdel Test](https://en.wikipedia.org/wiki/Bechdel_test) for the first time that anyone could recall.  There's far more to do here in terms of active inclusion, but we are truly seeing a more diverse TDC (community) _and_ TSC (leadership) than we have in past years.
